network-api/networkapi/tests.py

```diff
@@ -613,3 +616,52 @@ def test_fields_remain_untouched_and_wagtail_image_is_changed(self):
         del software_product_page_dict['image'], new_software_product_page_dict['image']
         del software_product_page_dict['cloudinary_image'], new_software_product_page_dict['cloudinary_image']
         self.assertDictEqual(software_product_page_dict, new_software_product_page_dict)
+
+
+class TestCreateWagtailImageUtility(TestCase):
+
+    def setUp(self):
+        self.image_path = abspath(join(dirname(__file__), '../media/images/placeholders/products/teddy.jpg'))
+
+    def create_new_image(self):
+        """A generic test to ensure the image is created properly."""
+        new_image = create_wagtail_image(
+            self.image_path,
+            image_name='fake teddy.jpg',
+            collection_name='pni products'
+        )
+        # Image was created
+        self.assertTrue(bool(new_image))
+        # Image has a collection and is in the proper collection
+        self.assertTrue(new_image.collection_id)
+        self.assertEqual(new_image.collection.name, 'pni products')
+
+    def test_empty_image_name_and_no_collection(self):
+        new_image = create_wagtail_image(
+            self.image_path,
+        )
+        self.assertEqual(new_image.title, 'teddy.jpg')
+        self.assertEqual(new_image.collection.name, 'Root')
+
+    def test_new_collection(self):
+        new_image = create_wagtail_image(
+            self.image_path,
+            image_name='fake teddy.jpg',
+            collection_name='brand new collection'
+        )
+        self.assertEqual(new_image.collection.name, 'brand new collection')
```

Comment:
Small nit, but this makes it impossible for typos to break this test in the future:
```suggestion
        collection_name = 'brand new collection'
        new_image = create_wagtail_image(
            self.image_path,
            image_name='fake teddy.jpg',
            collection_name=collection_name
        )
        self.assertEqual(new_image.collection.name, collection_name)
```